olive/passes/pytorch/ort_quant_utils.py

```diff
@@ -148,6 +148,7 @@ def __init__(self, bits, groupsize, infeatures, outfeatures, bias, *args, **kwar
         self.orig_fp_weight = None
```

Comment:
What is the motivation behind renaming this file?

Reply:
This quant utils can be leveraged by all the quantization method in QLLM, like AWQ, GPTQ, HQQ and etc.

Previous gptq utils sounds a bit narrow only to GPTQ method.

Reply:
Then just use `qaunt_utils.py`.  Having ORT in name may confuse folks.

Reply:
Done.